src/sourmash/commands.py

```diff
@@ -458,7 +458,12 @@ def search(args):
             notify("ERROR: cannot specify both --containment and --max-containment!")
             sys.exit(-1)
 
-    databases = sourmash_args.load_dbs_and_sigs(args.databases, query,
+    db_files = set(args.databases)
```

Comment:
this looks great! but I think making this a `list` would be better, because then you retain order (unlike sets).

Down below, you would use `db_files.extend` for a list, instead of `update` (which is what a set uses).